.travis.yml

```diff
@@ -0,0 +1,25 @@
+language: python
+python:
+- "3.6"
+# setup environment
+env:
+- DJANGO_VERSION=2.1.3
+
+services:
+- postgresql
+
+addons:
+ postgresql: '9.6'
```

Comment:
why do you have posgresql?was it part of the story

Reply:
I thought we would use Postgres for the backend? Should I remove it? The story was simply integrating Travis to our project. The specifics were left out.